Main.java

```diff
@@ -0,0 +1,16 @@
+public class Main {
+    public static void main(String[] args) {
+        Sort s = new Sort(10);
+        int [] array = s.generateArray(20);
+        s.displayArray();
+        s.bubbleSort(array);
+        s.removeDuplicate(array);
+        
+        
+        Sort s2 = new Sort(10);
+        int [] array2 = s2.generateArray(20);
```

Comment:
To było pierwsze zadanie, rozgrzewkowe, żeby sprawdzić Wasze umiejętności z GITa. Wszystko z GITem zrobiłeś ok. Ale ju- jeśli chodzi o testowanie swoich metod sortujących na samych intach to zrobiłeś kiepsko. Teraz nie musisz poprawiać, ale w następnych zadaniach weź pod uwagę, by przetestować różne skrajne wartości, które mogą zepsuć program. W stylu 0, liczba ujemna, jakiś float, jakaś duża liczba i tak dalej:)